app/actions/EventActions.js

```diff
@@ -386,13 +386,21 @@ export function unfollow(followId: number, eventId: number): Thunk<Promise<*>> {
     );
 }
 
-export function isUserFollowing(eventId: number, userId: number) {
-  return callAPI({
-    types: Event.IS_USER_FOLLOWING,
-    endpoint: `/followers-event/?target=${eventId}&follower=${userId}`,
-    method: 'GET',
-    meta: {
-      errorMessage: 'Henting av hendelse følgere feilet'
-    }
-  });
+export function isUserFollowing(
+  eventId: number,
+  userId: ?number
```

Comment:
Why do we still need `userId` here? The action is only used once, without sending `userId`

Reply:
Nice, I just assumed it was used elsewhere :upside_down_face: